packages/plans-ui/package.json

```diff
@@ -0,0 +1,51 @@
+{
+	"name": "@automattic/plans-ui",
+	"version": "1.0.0-alpha.0",
+	"description": "Domain picker.",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/plans-ui"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"types": "dist/types",
+	"scripts": {
+		"clean": "check-npm-client && npx rimraf dist ../../.tsc-cache/packages__plans-ui*",
+		"prepare": "check-npm-client && tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json && copy-assets",
+		"prepublish": "check-npm-client && yarn run clean",
+		"watch": "check-npm-client && tsc --project ./tsconfig.json --watch"
+	},
+	"dependencies": {
+		"@automattic/data-stores": "^1.0.0-alpha.1",
+		"@automattic/react-i18n": "^1.0.0-alpha.0",
+		"@wordpress/components": "^9.6.0",
+		"@wordpress/icons": "^2.0.0",
+		"classnames": "^2.2.6",
+		"lodash": "^4.17.15",
+		"react-modal": "3.11.2",
```

Comment:
I don't think we need `react-modal` or any other modal in the package. As we discussed yesterday, in wp-admin we'll use Modal from `@wordpress/components`. @yansern is looking into this so there will be probably already a pattern to follow for wrapping `PlansGrid` in a modal when landing in FSE.